### PR TITLE
Add default props to StyledIcon

### LINKED ISSUE #4: Set default props in StyledIcon component
### Description

- Add `#fff` as default color
- Add `24px` as default size
- Remove `prop-types`

### PR DESCRIPTION
**Pull request by rqbazan**
### Description

This PR:

- Adds a simple `prop` function
- Refactors `StyledIcon`
- Adds `commitizen`
- Closes #4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "icons": "node -r @std/esm generate-icons.js",
     "build": "babel ./src --out-dir dist ",
-    "prepublish": "yarn build"
+    "prepublish": "npm build",
+    "commit": "npx git-cz"
```
Comment (klujanrosas) [inline package.json]: you don't need `npx` :tada:

File: package.json [hunk 2]
```diff
@@ -20,13 +21,17 @@
...
+    "cz-conventional-changelog": "^2.1.0",
     "lodash": "^4.17.4",
     "pascal-case": "^2.0.1",
-    "prop-types": "^15.6.0",
     "svgo": "^1.0.3",
     "xml2js": "^0.4.19"
   },
-  "dependencies": {
-    "@riqra/riqra-icons": "^7.14.0"
+  "dependencies": {},
```
Comment (klujanrosas) [inline package.json]: Just remove the key entirely :tada:

File: src/styled-icon.js [hunk 3]
```diff
@@ -8,30 +9,31 @@ const sizes = {
...
-  display: ${props => props.display || 'inline-block'};
+  display: ${prop('display', 'inline-block')};
+
   flex: 0 0 auto;
+
+  width: ${resolveSize};
+  height: ${resolveSize};
+
+  *[stroke*='#'] {
+    stroke: ${prop('color', '#fff')}
```
Comment (klujanrosas) [inline src/styled-icon.js]: These two are the same, I believe you might want just make it a resolver.

Something like
```js
const resolveIconColor = prop('color, '#ffffff')
```

And reusing that.

File: src/styled-icon.js [hunk 4]
```diff
@@ -8,30 +9,31 @@ const sizes = {
   xl: '96px'
 }
 
+const resolveSize = ({ size = 'md' }) => sizes[size]
```
Comment (klujanrosas) [inline src/styled-icon.js]: Why not make it
```js
const resolveSize = prop('size', sizes.md)
```

**Comment by klujanrosas** [comment]
Also, this should not be merged to master.
It must be published to an `alpha` tag or smth

File: src/styled-icon.js (hunk 4 above)
Comment (rqbazan) [reply src/styled-icon.js]: That's not work, think about it:

```js
<Icon size='xs' />
```

**Pull request closed without merging**